qgroundcontrol.pro

```diff
@@ -307,7 +307,8 @@ HEADERS += \
     src/AutoPilotPlugins/APM/APMAirframeLoader.h \
     src/QmlControls/QGCImageProvider.h \
     src/QtLocationPlugin/QMLControl/QGCMapEngineManager.h \
-    src/PositionManager/PositionManager.h
+    src/PositionManager/PositionManager.h \
+    src/FirmwarePlugin/APM/ArduSubFirmwarePlugin.h
```

Comment:
The firmware plugin source is all sitting in a separate section towards the bottom of the file. Can you move this there so it is all together?